dev/com.ibm.ws.security.saml.sso_fat.jaxrs.config/fat/src/com/ibm/ws/security/saml/fat/jaxrs/config/IDPInitiated/RSSamlIDPInitiatedSSLConfigWithReconfigTests.java

```diff
@@ -73,24 +73,6 @@ public void RSSamlIDPInitiatedConfigTests_signatureMethodAlgorithm_SHA1() throws
         generalConfigTest(updatedRsSamlSettings, expectations, testSettings);
     }
 
-    /**
-     * Test purpose: - signatureMethodAlgorithm: SHA128 Expected results: - The
-     * SAML token should be successfully processed by JAX-RS.
-     *
-     * @throws Exception
-     */
-    @Test
-    public void RSSamlIDPInitiatedConfigTests_signatureMethodAlgorithm_SHA128() throws Exception {
-
-        RSSamlConfigSettings updatedRsSamlSettings = rsConfigSettings.copyConfigSettings();
-        RSSamlProviderSettings updatedProviderSettings = updatedRsSamlSettings.getDefaultRSSamlProviderSettings();
-        updatedProviderSettings.setSignatureMethodAlgorithm(CryptoUtils.MESSAGE_DIGEST_ALGORITHM_SHA128);
```

Comment:
Thanks for taking care of this quickly, Alex! Now that I'm looking at this, we should also remove the constant from CryptoUtils for SHA128 as it is invalid.